### PR TITLE
Add trace proto

### PR DESCRIPTION
**Pull request by ramonza**

### CODE REVIEW

File: proto/traceproto/doc.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
+// Copyright 2017, OpenCensus Authors
```
Comment (odeke-em) [inline proto/traceproto/doc.go]: s/2018/g

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[submodule "proto/opencensus-proto"]
+	path = proto/opencensus-proto
+	url = https://github.com/census-instrumentation/opencensus-proto
```
Comment (odeke-em) [inline .gitmodules]: Am not sure if we should be git submodule-ing stuff into Go as the convention is to use vendor. However this isn't a Go package plus using a git submodule will help us keep a separate git tree and update it when needed. Let's then add some instructions though for updating this submodule, if we go forward with it.

Comment (ramonza) [reply .gitmodules]: I thought it would be okay in this case since the submodule is only required for "go generate", not when using the library.

**Comment by rakyll** [comment]
We discussed earlier with @bogdandrutu not to publish generated code directly from this repo but https://github.com/census-instrumentation/opencensus-proto.

**Comment by ramonza** [comment]
@rakyll that makes sense, I'll add it there and close this PR

**Pull request closed without merging**